### ISSUE TITLE
Giotto

### ISSUE THREAD

**Issue by boegel** [vscentrum/vsc-software-stack#74]
* link to support ticket: [#2023010360000793](https://otrsdict.ugent.be/otrs/index.pl?Action=AgentTicketZoom;TicketID=105458)
* website: https://giottosuite.readthedocs.io
* installation docs: https://giottosuite.readthedocs.io/en/master/gettingstarted.html
* toolchain: `foss/2022a`
* easyblock to use: `RPackage`
* required dependencies:
  * [x] R
  * :various Python packages (see https://giottosuite.readthedocs.io/en/master/gettingstarted.html#part-2-python-giotto-requirements):
    ```
    pip3 install pandas python-igraph networkx python-louvain leidenalg scikit-learn smfishHmrf
    ```
* notes:
  * ...
* effort: *(TBD)*

**Comment by stevenvdb** [comment]
One of the needed Python packages is scanpy (it is not listed explicitly, but it is needed in some cases), so I will wait for https://github.com/easybuilders/easybuild-easyconfigs/pull/17142 to get merged. Apart from that I have an easyconfig for Giotto ready.

**Comment by boegel** [comment]
For `scanpy`, you can assume that https://github.com/easybuilders/easybuild-easyconfigs/pull/17160 will get merged soon.

So, use `scanpy` 1.9.1 as a dependency, and go ahead and open the PR. You can indicate that the PR depends on another PR being merged by using this in the PR description:

```
requires:
* #17160
```

GitHub will render that nicely, showing the PR title.

The CI tests will fail for the Giotto PR because of the missing `scanpy` easyconfig in the `develop` branch, but that's not a huge problem (we can easily retrigger the tests once the `scanpy` PR is merged).

**Comment by stevenvdb** [comment]
PR opened https://github.com/easybuilders/easybuild-easyconfigs/pull/17207

**Comment by boegel** [comment]
PR merged
